org.musicbrainz.Picard.json

```diff
@@ -140,14 +140,12 @@
         },
         {
             "name": "libdiscid",
-            "config-opts": [
-                "--disable-static"
-            ],
+            "buildsystem": "cmake-ninja",
```

Comment:
Their CMake build system doesn't seem to have an equivalent to the Autotools's `--disable-static`, but it seems to only build a shared library anyway.

In fact, their Autotools build system also only seems to build a shared library, so now I'm thinking I might just have cargo-culted this back in the days.

Looking at the build logs confirms they only build a shared library.

This is ready to be merged as soon as the test build finishes as far as I'm concerned.